src/rest-server/src/utils/frameworkConverter.js

```diff
@@ -1,46 +1,38 @@
+// Copyright (c) Microsoft Corporation
```

Comment:
could you put this file under `src/models/v2/job/`? it's not a common utility and is only used by v2/job related

Reply:
> could you put this file under `src/models/v2/job/`? it's not a common utility and is only used by v2/job related

Moved to `src/models/v2/utils`. The utility is also used by `v2/job-attempt` and `v2/task`.